driver/src/main/java/org/neo4j/driver/internal/SimpleRecord.java

```diff
@@ -57,7 +58,16 @@ public Value get( int fieldIndex )
     @Override
     public Value get( String fieldName )
```

Comment:
This same fix should apply to get( int ) as well, I think, so that we are consistent.